src/main/resources/db/migration/V4__create_role.sql

```diff
@@ -0,0 +1,19 @@
+CREATE SEQUENCE role_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+CREATE TABLE role (
+    id bigint DEFAULT nextval('role_id_seq'::regclass) NOT NULL,
+    name character varying(255) NOT NULL,
+    pairing_board_id bigint,
+    project_id bigint
```

Comment:
I don't think Roles can be attributed to projects. I would delete this.

Reply:
This is true, we forgot to remove that line after we changed how the feature worked, will do.